src/resolve/resolved-dns-answer.h

```diff
@@ -73,8 +73,6 @@ static inline bool dns_answer_isempty(DnsAnswer *a) {
         return dns_answer_size(a) <= 0;
 }
 
-void dns_answer_dump(DnsAnswer *answer, FILE *f);
```

Comment:
hmm, but this one is *sooo* useful when debugging stuff... i kept that for a reason...

Reply:
It looks like you didn't just keep it for that reason, you committed it for that reason:
https://github.com/systemd/systemd/commit/2615691003b9d73a92590b8250a54ad135e3a33b

looks useful to me, and spurious in this PR otherwise LGTM PR

Reply:
OK. I'll a comment in the source then. It's hard to remember what a commit from 2015 had in mind ;)